openquakeplatform/bin/deploy.sh

```diff
@@ -0,0 +1,383 @@
+}
+SECRET="$(key_create)"
+
+# create db pwd
+function passwd_create () {
+    python -c "import string ; import random
+def id_generator(size=8, chars=string.ascii_uppercase + string.digits):
+    return ''.join(random.choice(chars) for x in range(size))
+print id_generator()"
+}
+gem_db_pass="$(passwd_create)"
+
+function apache_tomcat_restart() {
+   sudo service apache2 restart
+   sudo service tomcat7 restart
+   sleep 220
+   echo "restart Apache/Tomcat"
+}
+
+function setup_postgres_once() {
+
+    sudo -u postgres createdb $GEO_DBUSER
+    sudo -u postgres createdb $GEO_DBUSER-imports
+
+    cat << EOF | sudo -u postgres psql
+    CREATE USER "geonode" WITH PASSWORD '$gem_db_pass';
+    GRANT ALL PRIVILEGES ON DATABASE "$GEO_DBUSER" to geonode;
+    GRANT ALL PRIVILEGES ON DATABASE "$GEO_DBUSER-imports" to geonode;
+EOF
+
+    sudo -u postgres psql -d $GEO_DBUSER -c 'CREATE EXTENSION postgis;'
+    sudo -u postgres psql -d $GEO_DBUSER -c 'GRANT ALL ON geometry_columns TO PUBLIC;'
+    sudo -u postgres psql -d $GEO_DBUSER -c 'GRANT ALL ON spatial_ref_sys TO PUBLIC;'
+
+    sudo -u postgres psql -d $GEO_DBUSER-imports -c 'CREATE EXTENSION postgis;'
+    sudo -u postgres psql -d $GEO_DBUSER-imports -c 'GRANT ALL ON geometry_columns TO PUBLIC;'
+    sudo -u postgres psql -d $GEO_DBUSER-imports -c 'GRANT ALL ON spatial_ref_sys TO PUBLIC;'
+
+    # add unaccent extension and icompare_unaccent function into postgres
+    cat << EOF | sudo -u postgres psql -d geonode                                                                                                                                                                                         
+    -- NOTE: originally deployed as migration, we realized that this sql script must
+    --       be executed for every new installation (devel or production).
+    --       The script is idempotent so we decided to keep the original migration script too
+    
+    DROP OPERATOR IF EXISTS =~@ (character varying, character varying);
+    DROP FUNCTION IF EXISTS icompare_unaccent(character varying, character varying);
+    DROP EXTENSION IF EXISTS unaccent;
+    
+    CREATE EXTENSION unaccent;
+    CREATE FUNCTION icompare_unaccent(character varying, character varying) RETURNS boolean
+        AS 'SELECT upper(unaccent(\$1)) LIKE upper(unaccent(\$2));'
+        LANGUAGE SQL
+        IMMUTABLE
+        RETURNS NULL ON NULL INPUT;
+   
+    CREATE OPERATOR =~@ (
+        LEFTARG = character varying,
+        RIGHTARG = character varying,
+        PROCEDURE = icompare_unaccent,
+        NEGATOR = !=~@
+    );
+EOF
+
+    # insert line in pg_hba.conf postgres
+#     sudo sed -i '1 s@^@local  all             '"$GEO_DBUSER"'             md5\n@g' /etc/postgresql/9.5/main/pg_hba.conf
+#    if [ "$DEVEL_DATA" ] || [ "$DATA_PROD" ]; then
+#        PG='/32'
+#    fi
+#    sudo sed -i '2 s@^@host  all    '"$GEO_DBUSER"'         '"$LXC_IP""$PG"'             md5\n@g' /etc/postgresql/9.5/main/pg_hba.conf
```

Comment:
What to do with this commented block of code ?

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/66/commits/67d552d3dc00a3244a31e4b5d5d95877f2225732